### DISCUSSION TITLE
Error: sha256 mismatch

### DISCUSSION THREAD

**Discussion by airvair** [Homebrew/discussions#5404] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: e0bc557e7b991cb23583679e1cf1c8a92b793aeb
Last commit: 6 days ago
Core tap JSON: 26 May 15:50 UTC
Core cask tap JSON: 26 May 15:50 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.1 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.1/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

~ ❯ brew install --cask stats                                      
==> Downloading https://github.com/exelban/stats/releases/download/v2.10.15/Stat
==> Downloading from https://objects.githubusercontent.com/github-production-rel
######################################################################### 100.0%
Error: SHA256 mismatch
Expected: 761b8546645806a881e7ade245301ea5257953621fbf43a95e95883ac72f6f62
  Actual: aba1504031e26fbf6bd29c0d962d3c7d97f35a5f1ecded5233554d42503674ca
    File: /Users/preston/Library/Caches/Homebrew/downloads/8f040f5dcdcc989dbe4edab24e99b08f14402d5c46ad9bb66d459ac50988341e--Stats.dmg
To retry an incomplete download, remove the file above.

---------------------------------------

Installing programs on my m2 macbook air and this issue comes up. I have installed stats using homebrew before on this laptop. I am not sure why I am getting this issue.

### Relevant casks

stats

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match

**Reply by airvair** [reply]
~ ❯ brew info stats                                                                13:28:30
==> stats: 2.10.15 (auto_updates)
https://github.com/exelban/stats
Not installed
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/s/stats.rb
==> Name
Stats
==> Description
System monitor for the menu bar
==> Artifacts
Stats.app (App)
==> Analytics
install: 6,891 (30 days), 23,636 (90 days), 65,280 (365 days)


The latest homebrew version is 2.10.15 and that is also the same on stats gh page. https://github.com/exelban/stats/releases/tag/v2.10.15

I think this means it is up to date right?

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/174951
